crates/router/src/connector/cybersource.rs

```diff
@@ -104,6 +104,13 @@ impl ConnectorCommon for Cybersource {
             .map(|det| format!("{} : {}", det.field, det.reason))
             .collect::<Vec<_>>()
             .join(", ");
+
+        let error_message = if res.status_code == 401 {
+            "Authentication Error from the connector"
```

Comment:
Maybe this can be made a constant and used for other connectors as well wherever applicable